src/components/ArmorModal.js

```diff
@@ -0,0 +1,357 @@
+import React, { Component } from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import {
+  Card, withStyles, CardHeader, CardContent,
+  Table, TableHead, TableRow, TableBody, TableCell,
+  Tooltip, CardMedia, CardActions, Button, Typography,
+  FormControl, Select, MenuItem, InputLabel, OutlinedInput,
+  IconButton,
+} from "@material-ui/core";
+import { Close } from "@material-ui/icons";
+import { Formik } from "formik";
+import {
+  Radar, RadarChart, PolarGrid, PolarAngleAxis, PolarRadiusAxis,
+} from "recharts";
+import _ from "lodash";
+import { armorToLoadout } from "store/ducks/Loadouts";
+import { SlidingModal, Panel } from "components/layout";
```

Comment:
you are mixing component with layout, layout is pretty much container, row, all the boxes of a page, 

you should have modal, panel folder for these